elliottlib/find_cve_flaws.py

```diff
@@ -0,0 +1,76 @@
+import requests
+import ssl
+from elliottlib import constants, bzutil
+from requests_kerberos import HTTPKerberosAuth
+
+
+def get_attached_tracker_bugs(bzapi, advisory_id):
+    return bzapi.getbugs([
+        bug['id']
+        for bug in get_all_attached_bugs(advisory_id)
+        if is_tracker_bug(bug)
+    ])
+
+
+def get_all_attached_bugs(advisory_id):
+    return [bug['bug'] for bug in get_advisory(advisory_id)['bugs']['bugs']]
+
+
+def is_tracker_bug(bug):
+    return 'Security' in bug['keywords'] and 'SecurityTracking' in bug['keywords']
+
+
+def get_advisory(advisory_id):
```

Comment:
I suggest we share this optimization in https://github.com/openshift/elliott/pull/163/files#diff-2da8b2e6b9ce21ce20f06eebc4842ab69122abe29406b251f2326f37bbee16e9R34-R39

Reply:
neat! :+1: done